proto/warden/warden/v1beta3/tx.proto

```diff
@@ -178,6 +179,7 @@ message MsgNewKeyRequest {
   uint64 keychain_id = 3;
   KeyType key_type = 4;
   uint64 rule_id = 5;
+  string max_fees = 6;
```

Comment:
can we rename it to "max_keychain_fees" so it's not confused with the normal tx fees?

Reply:
✅